about.php

```diff
@@ -29,28 +29,23 @@
 	</div>
 	<div class="heritage">
 		<h3>Our Heritage</h3>
-		<p>We are humbled by the success of our company thus far. The name Starbulls was created by one of the founders:
-			Will. He came up with this insane concept of a restaurant that serves wings and coffee.
-		<p>
+		<p>We are humbled by the success of our company thus far. The name StarBulls was created by Will, 
+			one of our founders. Fueled by a continual flow of caffeine and blue cheese, Will opened StarBulls in Febuary of 2021.
+		</p>
 		<p>How do we measure success? Simply: we started from the bottom, now we are here! Seriously,
-			look at our site! A beauty to say the least.</p>
+			look at our site! A beauty to say the least.
+		</p>
 		<p>Just a week ago, StarBulls did not exist. It has taken hours of dedication to get this
-			company up
-			on the server. Hours of frustration, trying to figure out why the code looks different on
-			our
-			screens. Hours of trial and error as we figured out where we went wrong. One thing the
-			founders
-			of StarBulls has known from the beginning is: Quitters Never Win. And Winners Never Quit. At
-			StarBulls we are
-			Winners.
+			company up on the server. Hours of frustration, trying to figure out why the code looks different on
+			our screens. Hours of trial and error as we figured out where we went wrong. One thing the
+			founders of StarBulls has known from the beginning is: Quitters Never Win. And Winners Never Quit. At
```

Comment:
Should this be:
founders of StarBulls have known

Reply:
Probably 'StarBulls knew from'